ISHPermissionKit/Requests/ISHPermissionRequest+All.h

```diff
@@ -24,6 +27,18 @@
  *  @return An instance of the appropriate ISHPermissionRequest subclass
  *          for the given category.
  */
-+ (nonnull ISHPermissionRequest *)requestForCategory:(ISHPermissionCategory)category;
++ (ISHPermissionRequest *)requestForCategory:(ISHPermissionCategory)category;
 
+/**
+ *  Check the permission state for a set of permissions.
+ *
+ *  @param categories Set of boxed ISHPermissionCategory to check for permission state.
+ *
+ *  @return A set of boxed ISHPermissionCategory values for which the
+ *          user has granted permissions among the set of
+ *      categories given as an argument.
+ */
++ (NSSet<NSNumber *> *)grantedPermissionsForCategories:(NSSet<NSNumber *> *)categories;
```

Comment:
I think what we really wanted to know is which permissions would still trigger a prompt. We can keep this method, but we'd still need something that aligns with `setupRequestablePermissionsCategoriesFromArray:` from `ISHPermissionsViewController`. We'd probably want to move the internal implementation to this file, and call it in the permissions VC setup.